lib/install/management/appliance.go

```diff
@@ -599,6 +599,7 @@ func (d *Dispatcher) createAppliance(conf *config.VirtualContainerHostConfigSpec
 				"CS_PATH=" + settings.ClusterPath,
 				"POOL_PATH=" + settings.ResourcePoolPath,
 				"DS_PATH=" + conf.ImageStores[0].Host,
+				"VC_KEEPALIVE=20s",
```

Comment:
I find that we have different keep_alive values in our code here and there. Why do you set it to `20s` here?